NEWS.rst

```diff
@@ -0,0 +1,130 @@
+    wrappers.
+  - Better ``repr()`` for ``epsilon.structlike.record`` added.
+  - ``epsilon.juice`` now uses ``twisted.internet.ssl`` instead of
+    ``epsilon.sslverify``.
+
+0.5.9 (2008-01-18):
+
+0.5.8 (2007-11-27):
+  - ``extime.Time.asHumanly`` no longer shows a time of day for all-day
+    timestamps.
+
+0.5.7 (2007-04-27):
+  - ``view.SlicedView`` added, allowing slicing and indexing of large
+    sequences without copying.
+
+0.5.6 (2006-11-20):
+  - Added a ``--quiet`` option to *Epsilon's* *certcreate* and use it in a few
+    unit tests to avoid spewing garbage during test runs.
+
+0.5.5 (2006-10-21):
+  - ``extime.Time`` now accepts RFC2822-like dates with invalid fields: it
+    rounds them to the nearest valid value.
+
+0.5.4 (2006-10-17):
+  - ``extime.Time`` now accepts RFC2822-like dates with no timezone.
+
+0.5.3 (2006-09-20):
+  - ``structlike.Record`` now raises ``TypeError`` on unexpected args.
+
+0.5.2 (2006-09-12):
+  - ``extime.Time`` now avoids ``time_t`` overflow bugs.
+
+0.5.1 (2006-06-22):
+  - Added hotfix for ``twisted.test.proto_helpers.StringTransport``.
+
+0.5.0 (2006-06-12):
+  - Replaced ``'%y'`` with ``'%Y'`` in ``Time.asHumanly`` output - the year is
+    now four digits, rather than two.
+  - Added new ``epsilon.structlike`` functionality for simple record.
+  - All uses of defer.wait and deferredResult were removed from the tests.
+  - Added ``epsilon.juice``, an asynchronous messaging protocol slated for
+    inclusion in *Twisted*.  Improved a few features, such as the ``repr`` of
+    JuiceBox instances.  This was moved from *Vertex*.
+  - Added ``epsilon.sslverify``, a set of utilities for dealing with
+    *PyOpenSSL* using simple high-level objects, performing operations such as
+    signing and verifying certificates. This was also moved from *Vertex*, and
+    slated for inclusion in *Twisted*.
+  - Added ``epsilon.spewer``, a prettier version of the spewer in
+    ``twisted.python.util``.
+  - Added "benchmark" tool for measuring and reporting run-times of python
+    programs.
+
+0.4.0 (2005-12-20):
+  - Disabled crazy ``sys.modules`` hackery in ``test_setuphelper``.
+  - Added module for creating a directory structure from a string template.
+  - Added support for *now* to ``Time.fromHumanly``.
+  - Added a structured "hotfix" system to abstract and formalize monkey
+    patches and version testing logic away from code which requires it.
+
+0.3.2 (2005-11-05):
+  - Added automatic support for *Twisted* plugins to ``autosetup``.
+
+0.3.1 (2005-11-02):
+  - Removed bogus dependency on *Axiom*.
+
+0.3.0 (2005-11-02):
+  - Added ``SchedulingService``, an ``IService`` implementation, to
+    ``epsilon.cooperator``.
+  - Added ``autosetup``, a utility to actually include files in *distutils*
+    releases, to ``epsilon.setuphelper``.
+
+0.2.1 (2005-10-25):
+  - Added ``short`` to ``epsilon.versions.Version``.
+  - fixed *setup.py* to use ``epsilon.version.short`` rather than static
```

Comment:
The casing seems to be incorrect at the start of this line. However, it was the same prior to this branch.